src/libs/workspace-utils.ts

```diff
@@ -164,6 +165,31 @@ export const regionConstraintMessage = (workspace: BaseWorkspace): string | unde
     : `Workspace storage and compute resources must remain in the following region(s): ${regions.join(', ')}.`;
 };
 
+const isGroupConstraintPolicy = (policy: WorkspacePolicy): boolean => {
+  return policy.namespace === 'terra' && policy.name === 'group-constraint';
+};
+
+export const hasGroupConstraint = (workspace: WorkspaceWrapper): boolean => {
+  return (workspace.policies || []).some(isGroupConstraintPolicy);
+};
+
+export const groupConstraintMessage = (workspace: WorkspaceWrapper): string | undefined => {
+  if (!hasGroupConstraint(workspace)) {
+    return undefined;
+  }
+
+  const groupConstraintPolicies = (workspace.policies || []).filter(isGroupConstraintPolicy);
+  const requiredGroups = _.flow(
+    _.flatMap((policy: WorkspacePolicy) => policy.additionalData.map((d) => d.group)),
+    _.uniq,
+    _.sortBy(_.identity)
+  )(groupConstraintPolicies);
+  return `Data Access Controls add additional permission restrictions to a workspace. These were added when you imported data from a controlled access source. All workspace collaborators must also be members of the following ${pluralize(
+    'group',
+    requiredGroups.length
+  )}: ${requiredGroups.join(', ')}.`;
```

Comment:
@bethsheets here's the PR with the text for the group constraint / data access controls notice on the workspace dashboard.

Reply:
@nawatts @cahrens I know it is not ideal, but we could not agree on what the policy name should be (AnVIL data access troubleshooting is complex) and so it was moved to post-MVP because our first priority is making sure the end to end flow works. Identi-team is leading the initiative to identify the right data access control terms and what a troubleshooting journey looks like.  We don't have AnVIL users on Azure quite yet, so we have time to make this better. We agreed on being generic for the time being "data access controls are present"

Reply:
I'm confused... it sounds like we've decided on a term: "data access controls". What prevents being able to show the groups required?